lib/intf.js

```diff
@@ -1895,6 +1692,11 @@ export default function (self, ctor) {
     },
   });
   Object.defineProperty(self.intf, 'selections', {
+    get: function () {
+      return self.getObsoleteSelectionMatrix();
```

Comment:
Should we warn the user here, of accessing a soon-to-be-deprecated version of `selections` ?